full-stack-tests/core/src/frontend/hub/MapLayerSettingsService.test.ts

```diff
@@ -16,7 +16,7 @@ const testIModelName = "ReadOnlyTest";
 
 chai.should();
 
-describe("MapLayerSettingsService (#integration)", () => {
+describe.skip("MapLayerSettingsService (#integration)", () => {
```

Comment:
Thanks. Can you add a comment indicating why it's skipped? Caleb et al are working on getting all the integration tests passing and un-skipped, so they'll want to know.

Reply:
Added